CDS/WebContent/WEB-INF/jsps/layout/head.jsp

```diff
@@ -24,6 +24,13 @@
 <script src="${pageContext.request.contextPath}/js/bootstrap.bundle.min.js"></script>
 <script src="${pageContext.request.contextPath}/js/adminlte.min.js"></script>
 <script src="${pageContext.request.contextPath}/js/theme.js"></script>
+<script>
+function logout() {
```

Comment:
What does this magic do?

Reply:
Nothing, at the moment. :) A hack I saw online to log out of Basic auth in a web browser is to try logging in with an incorrect user in the background, was starting to try that and haven't finished.